src/Gui/PreferencePages/DlgSettingsEditor.cpp

```diff
@@ -280,13 +281,13 @@ void DlgSettingsEditor::loadSettings()
 
     // Restores the color map
     ParameterGrp::handle hGrp = WindowParameter::getDefaultParameter()->GetGroup("Editor");
-    for (QVector<QPair<QString, unsigned int>>::Iterator it = d->colormap.begin();
-         it != d->colormap.end(); ++it) {
-        auto col = static_cast<unsigned long>((*it).second);
-        col = hGrp->GetUnsigned((*it).first.toLatin1(), col);
-        (*it).second = static_cast<unsigned int>(col);
+    for (auto &[textType, textColor]: d->colormap)
```

Comment:
The linter wants whitespace around the :
```suggestion
    for (auto &[textType, textColor] : d->colormap)
```